abc.templates/infra/contents/main.tf

```diff
@@ -41,3 +41,11 @@ module "REPLACE_MODULE_NAME" {
   leech_bucket_name     = local.leech_bucket_name
   leech_bucket_location = local.leech_bucket_location
 }
+
+module "REPLACE_MODULE_NAME_guardian_iam" {
+  source = "git::https://github.com/abcxyz/github-metrics-aggregator.git//terraform?ref=v0.0.7"
```

Comment:
will bump version in follow up

Reply:
Rather than having this be a module, should we just create an `iam.tf` file, then you can depend on the `google_project_iam_member.github_metrics_guardian_iam` instead. Since that resource depends on the custom roles, it should "just work".

Also, do we want to use the word Guardian here for resource names? There could be a possibility that people arent using Guardian which could be confusing.